tests/Cookie/SessionCookieJarTest.php

```diff
@@ -67,9 +67,19 @@ public function testPersistsToSession()
         // Load the cookieJar from the file
         $jar = new SessionCookieJar($this->sessionVar);
 
-        // Weeds out temporary and session cookies
-        $this->assertEquals(2, count($jar));
+        if($testSaveSessionCookie) {
```

Comment:
@mtdowling Maybe you should enable https://styleci.io/ on this repo so you don't have to review cs manually?